utils/compression.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+from __future__ import with_statement
+
+import logging
+import os
+import subprocess
+from string import Template
+
+logger = logging.getLogger(__name__)
+
+
+class Compressor(object):
+    """
+    Compress files.
+    """
+
+    def __init__(self, input=None, output=None, debug=False):
+        """
+        Initialize the compressor.
+
+        Args:
+            input: the file to compress
+            output: the output file
+        """
+        self.input = input
+        if not os.path.exists(self.input):
+            raise IOError('Cannot find input file for this task.')
+        self.output = output
+        if not self.output:
+            # create output path from input path.
+            self.output = self.input + '.bz2'
+        self.debug = debug
+        self.cmd = Template('bzip2 -c $input > $output')
+
+    def compress(self, ):
```

Comment:
This got pulled over from one of the classes I was following, so I was thinking it was local style? (That also explains the string formatting and likely a variety of other things you'll notice.)

For diff-ability, I do like the newline-expanded form with consistently trailing commas, as it reduces noise, but that's not really applicable here:

```python
def compress(
  self,
):
```